CHANGELOG.md

```diff
@@ -1,5 +1,112 @@
 # vscode-powershell Release History
 
+## v2.0.0-preview.2
+### Wednesday, March 27, 2019
+
+### Highlights
+
+* `Write-Progress` work in the integrated console ⏰
+* Support for [PSScriptAnalyzer 1.18](https://github.com/PowerShell/PSScriptAnalyzer/releases/tag/1.18.0) 📝
+* The ability to debug any runspace in any process 🔎
+* PSReadLine enabled by default on Windows 🎨
+* (Bug fix!) You can open untitled workspaces again! 🐛☠️
```

Comment:
Untitled "workspaces" or untitled documents?

Reply:
Workspaces is right here. It's often called an Untitled folder